src/components/SFuelBadge/SFuelBadge.tsx

```diff
@@ -7,10 +7,9 @@ import Tooltip from '@mui/material/Tooltip';
 import localStyles from './SFuelBadge.scss';
 
 
-
 const BadgeStates = {
   success: {
-    tooltip: 'Ok text',
+    tooltip: 'Everything is good',
```

Comment:
If this is after sFUEL is attained, suggest something more direct:

- Gas Token Received
- sFUEL Availalbe